static/app/components/profiling/flamegraphZoomView.tsx

```diff
@@ -328,6 +335,7 @@ function FlamegraphZoomView({
     gridRenderer,
     selectedNode,
     hoveredNode,
+    selectedFrameRenderer,
```

Comment:
@Zylphrex I wonder if this had something to do with that selectedNode not clearing (I dont think so, but would have been annoying to track if we persisted any state in the renderer)